src/transformState.js

```diff
@@ -6,6 +6,28 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action['extraData']);
```

Comment:
if you know the exact name of the variable you may use `.` to access the property
```suggestion
        Object.assign(state, action.extraData);
```

Reply:
done